app/src/main/java/us/shandian/giga/ui/fragment/MissionsFragment.java

```diff
@@ -208,6 +208,14 @@ public boolean onOptionsItemSelected(MenuItem item) {
         }
     }
 
+    public void areYouSureToDeleteAllFiles(){
+        AlertDialog.Builder prompt = new AlertDialog.Builder(mContext);
+        prompt.setTitle("Are you sure you want to delete all your files ?");
+        prompt.setPositiveButton("NO",  null);
+        prompt.setNegativeButton(R.string.yes, (dialog, which) -> mAdapter.clearFinishedDownloads(true));
```

Comment:
Is this correct? The positive button is `NO` and the negative is `R.string.yes` 

Also use `R.string.no` instead of `NO`

Reply:
R.string.no does not exist, do I have to create it?

Reply:
ehhh. There should be a string that already used, which you can reuse. This is definetly not the first place that needs a `no` string. You can check what string is behind `R.string.cancel`

Reply:
apparently "no" alone does not exist and R.string.cancel just write cancel

Reply:
Is there anything wrong with using `R.string.cancel` instead of adding a "no" string?